### DISCUSSION TITLE
Your issue yaml is broken

### DISCUSSION THREAD

**Discussion by yuvaltassa** [pradyunsg/furo#833] [Potential Issue] [answered]
I remembered how awesome your GH issue template yaml is and wanted to that for our project and noticed it was broken (you changed it last month and forgot a `label` line). GitHub is telling you what the error is https://github.com/pradyunsg/furo/blob/main/.github/ISSUE_TEMPLATE/issue.yml

**Comment by pradyunsg** [comment, marked as answer]
🤦🏽 

Fixed it. Thanks @yuvaltassa! ^.^
